### PR TITLE
📖  Updated the community links

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

     Updated the community links to be more accessible. Took the idea from https://github.com/tektoncd/community/blob/main/working-groups.md#chains

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by azeemshaikh38 — APPROVED** [review]
LGTM. Will leave it to @olivekl for the final approval.

**Comment by naveensrinivasan** [comment]
> LGTM. Will leave it to @olivekl for the final approval.

Thanks!

**Comment by naveensrinivasan** [comment]
@laurentsimon  I included your name in this. I completely forgot to get your approval. Can you please review it?

File: README.md [hunk 1]
```diff
@@ -391,23 +391,23 @@ this project in the
...
-for the schedule and meeting invitations. The meetings [happen biweekly](https://calendar.google.com/calendar/embed?src=s63voefhp5i9pfltb5q67ngpes%40group.calendar.google.com&ctz=America%2FLos_Angeles).
 
-The biweekly meeting minutes document can be found [here](https://docs.google.com/document/d/1dB2U7_qZpNW96vtuoG7ShmgKXzIg6R5XT5Tc-0yz6kE/edit#) (join ossf-scorecard-dev@googlegroups.com for access).
 
-[Join](https://support.google.com/groups/answer/1067205?hl=en) our email groups:
+| Artifact                      | Link                       |
+| --------------------------    | -------------------------- |
+| Scorecard Dev Forum           | [ossf-scorecard-dev@](https://groups.google.com/g/ossf-scorecard-dev) |
+| Scorecard Announcements Forum | [ossf-scorecard-announce@](https://groups.google.com/g/ossf-scorecard-announce) |
+| Community Meeting VC          | [https://zoom.us/j/98835923979](https://zoom.us/j/98835923979) |
```
Comment (laurentsimon) [inline README.md:400]: is it standard to provide easy-to-crawl/parse meeting links this way? No fear of zoom-bombing or the like? I don't think we can do anything about it, but if you come across a solution, let's use it in the future

**Review by laurentsimon — APPROVED** [review]
LGTM, thanks!

**Comment by olivekl** [comment]
My only concern would be including the Zoom link, which Laurent mentioned as well. Otherwise, everything LGTM. Thanks for these improvements!

**Comment by naveensrinivasan** [comment]
> My only concern would be including the Zoom link, which Laurent mentioned as well. Otherwise, everything LGTM. Thanks for these improvements!

I updated the Zoom link title not to have the actual link. Thanks

**Comment by laurentsimon** [comment]
> > My only concern would be including the Zoom link, which Laurent mentioned as well. Otherwise, everything LGTM. Thanks for these improvements!
> 
> I updated the Zoom link title not to have the actual link. Thanks

Maybe we could say "See calendar invite below"

**Pull request merged**